core/common/src/main/scala/org/typelevel/otel4s/ContextPropagators.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s
+
+import cats.Applicative
+
+trait ContextPropagators[F[_]] {
+  def textMapPropagator: TextMapPropagator[F]
+}
+
+object ContextPropagators {
+  def noop[F[_]: Applicative, C]: ContextPropagators[F] =
```

Comment:
The type param `C` is not used anywhere.

```suggestion
  def noop[F[_]: Applicative]: ContextPropagators[F] =
```